#! /bin/sh /usr/share/dpatch/dpatch-run

## 18_vdr-plugin-sky-fix.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix building of the sky plugin

@DPATCH@
diff -urNad vdr-1.3.32/PLUGINS/src/sky/sky.c /tmp/dpep.s0biey/vdr-1.3.32/PLUGINS/src/sky/sky.c
--- vdr-1.3.32/PLUGINS/src/sky/sky.c	2004-12-26 13:45:22.000000000 +0100
+++ /tmp/dpep.s0biey/vdr-1.3.32/PLUGINS/src/sky/sky.c	2005-09-11 22:23:27.825058752 +0200
@@ -13,6 +13,7 @@
 #include <unistd.h>
 #include <vdr/plugin.h>
 #include <vdr/sources.h>
+#include <vdr/device.h>
 
 static const char *VERSION        = "0.3.3";
 static const char *DESCRIPTION    = "Sky Digibox interface";
